src/Module/TechnicalInformation/Fetch/TechnicalInformation.php

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+namespace Pokerphace\Domain\Module\TechnicalInformation\Fetch;
+
+class TechnicalInformation
+{
+    private string $projectName;
+
```

Comment:
Is it necessary this empty lines here?

Reply:
Maybe not, I usually just place a space by default between each property

Reply:
Should I remove the spaces?